apps/avatax/src/modules/saleor/order.test.ts

```diff
@@ -0,0 +1,96 @@
+import { OrderCancelledPayload } from "../webhooks/payloads/order-cancelled-payload";
+import { SaleorCancelledOrderEvent } from "./order";
+import { expect, it, describe } from "vitest";
+import { OrderCancelNoAvataxIdError, OrderCancelPayloadOrderError } from "./order-cancel-error";
+
+describe("SaleorCancelledOrderEvent", () => {
+  const validPayload = {
+    order: {
+      channel: {
+        id: "channel-id",
+        slug: "channel-slug",
+      },
+      id: "order-id",
+      avataxId: "avatax-id",
+    },
+    recipient: {
+      privateMetadata: [
+        {
+          key: "key",
+          value: "value",
+        },
+      ],
+    },
+    __typename: "OrderCancelled",
+  } as OrderCancelledPayload;
+
+  it("should create a SaleorCancelledOrderEvent from a valid payload", () => {
+    const result = SaleorCancelledOrderEvent.create(validPayload);
+
+    expect(result.isOk()).toBe(true);
+
+    if (result.isOk()) {
+      expect(result.value).toBeInstanceOf(SaleorCancelledOrderEvent);
+    }
+  });
+
+  /*
```

Comment:
**Suggestion:** do we want to keep this commented?

I think `it.skip` will be better with link to linear issue 😄

Reply:
no no, I was trying to test how the logs will look like in datadog if we use `cause`, the code will be uncommented 😛